README.rst

```diff
@@ -0,0 +1,164 @@
+Jina Documentation
```

Comment:
I'm pretty sure GitHub requires the README is markdown not .rst 

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes

Reply:
Yessss, i will change this